std/zip.d

```diff
@@ -302,9 +323,9 @@ final class ZipArchive
     private uint _totalEntries;
     private bool _isZip64;
     static const ushort zip64ExtractVersion = 45;
-    static const int digiSignLength = 6;
-    static const int eocd64LocLength = 20;
-    static const int eocd64Length = 56;
+    deprecated("Use digitalSignatureLength instead") static const int digiSignLength = 6;
+    deprecated("Use zip64EndOfCentralDirLocatorLength instead") static const int eocd64LocLength = 20;
+    deprecated("Use zip64EndOfCentralDirLength instead") static const int eocd64Length = 56;
```

Comment:
These could also be aliases to the new definitions, but this is fine too.

I think according to https://github.com/dlang/DIPs/blob/master/DIPs/accepted/DIP1013.md#summary-of-the-steps-in-the-normal-deprecation-process this should have something like "will be removed in 2.098.0" so we know when we can remove them.

Reply:
2.093.0 is sufficient as there are two releases a year, isn't it?

Reply:
I think we decided to use a fixed number of major versions as we can't really predict when releases will occur.